src/components/button.js

```diff
@@ -2,18 +2,20 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = props => {
-  const { name } = props;
+  const { name, customClass } = props;
   return (
-    <button type="button">{name}</button>
+    <button className={`button ${customClass}`} type="button">{name}</button>
   );
 };
 
 Button.propTypes = {
   name: PropTypes.string,
+  customClass: PropTypes.string,
 };
 
```

Comment:
this component needs an incoming prop called `wide`, which is a boolean(confirm this with propTypes)
[ see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorAndWideProps.png) ]

Reply:
All requested changes made.